packages/utils-authorizer/src/index.ts

```diff
@@ -0,0 +1,91 @@
+import { APIGatewayTokenAuthorizerEvent, APIGatewayAuthorizerResult } from 'aws-lambda'
+import * as jwt from 'jsonwebtoken'
+import { JwtRsaVerifier, CognitoJwtVerifier } from 'aws-jwt-verify'
+
+export const authorizerHandler =
+  (customChallenge?: (event: APIGatewayTokenAuthorizerEvent, decodedToken: jwt.Jwt) => Promise<void | Error>) =>
+  async (event: APIGatewayTokenAuthorizerEvent): Promise<APIGatewayAuthorizerResult> => {
```

Comment:
Some comments in here would be useful just so anybody looking at it can understand why it's doing what it's doing